plugin/pkg/skplug/dispatcher/dispatcher.go

```diff
@@ -0,0 +1,128 @@
+package dispatcher
```

Comment:
Did you deleted the other one, `plugindispatcher`?

Reply:
Thank you for noticing this. Deleted, see commit  1e4a48d16e7047fe9be0f01eae61c2b297c481b8